### PR TITLE
Fix a correct onGLFWCharCallback

### PR DESCRIPTION
**Pull request by cpascal**

### CODE REVIEW

**Comment by minggo** [comment]
[ci rebuild]

**Comment by minggo** [comment]
@cpascal
What's the issue?

**Comment by cpascal** [comment]
In desktop (Microsoft Windows), it can't input Unicode(like CJK) characters.

File: cocos/platform/desktop/CCGLViewImpl.cpp [hunk 1]
```diff
@@ -650,11 +651,19 @@ void GLViewImpl::onGLFWKeyCallback(GLFWwindow *window, int key, int scancode, in
             IMEDispatcher::sharedDispatcher()->dispatchDeleteBackward();
         }
     }
+    if (GLFW_RELEASE != action && g_keyCodeMap[key] == EventKeyboard::KeyCode::KEY_BACKSPACE)
```
Comment (zilongshanren) [inline cocos/platform/desktop/CCGLViewImpl.cpp:654]: This modification had introduced a new bug. When I pressed the backspace key, it deleted twice. I will fix this issue in another PR.

**Pull request merged**